packages/core/examples/buttonsExample.tsx

```diff
@@ -16,16 +16,18 @@ import { IntentSelect } from "./common/intentSelect";
 export interface IButtonsExampleState {
     disabled?: boolean;
     intent?: Intent;
+    isLoading?: boolean;
```

Comment:
I'd go for `loading` -- it's simpler and more consistent with other `boolean` usage in this interface